crates/rolldown/src/utils/parse_to_ast.rs

```diff
@@ -50,6 +52,10 @@ pub fn parse_to_ast(
     ParseType::Jsx => todo!(),
     ParseType::Ts => todo!(),
     ParseType::Tsx => todo!(),
+    ParseType::Text => OxcCompiler::parse(
```

Comment:
You should not touch the `ParserType`. See how rolldown deal with `json`.

Reply:
has reverted and changed~